crates/autopilot/src/database/competition.rs

```diff
@@ -17,9 +17,9 @@ use {
 
 #[derive(Clone, Debug)]
 pub enum ExecutedFee {
-    /// Fee is calculated by the solver and known upfront (before the settlement
-    /// is finalized).
-    Solver(U256),
+    /// Fee is taken from the signed order and known upfront (before the
+    /// settlement is finalized).
```

Comment:
Can we add here that this is the full fee with any subsidies removed?

Reply:
Added